### ISSUE TITLE
localhost:50080/ords/ returns 404 Not Found

### ISSUE THREAD

**Issue by rcordoni** [fuzziebrain/docker-apex-stack#3]
I am able to create the container without apparent errors but I can't browse to the Apex website with localhost:50080/ords/.  I am using pretty stock mysettings.env with just a couple of password changes:

ORACLE_SID=XE
ORACLE_PDB=XEPDB1
ORACLE_PWD=********
APEX_ADMIN_EMAIL=***@***.ca
APEX_ADMIN_PWD=ApexRu1Z
APEX_PUBLIC_USER_PWD=Oracle18_1
APEX_LISTENER_PWD=Oracle18_2
APEX_REST_PUBLIC_USER_PWD=Oracle18_3
ORDS_PUBLIC_USER_PWD=Oracle18_4
INSTALL_FILE_APEX=apex_19.1_en.zip
INSTALL_FILE_ORDS=ords-18.4.0.354.1002.zip
INSTALL_FILE_JAVA=jdk-8u201-linux-x64.tar.gz
DOCKER_ORDS_PORT=50080
DOCKER_EM_PORT=55500
DOCKER_DB_PORT=51521
DB_VERSION=18.4.0
DB_EDITION=xe

Inside the container, using curl to test:
curl -I http://localhost:8080/ords
HTTP/1.1 302 Found
Date: Fri, 05 Apr 2019 21:20:30 GMT
Location: http://localhost:8080/ords/
Content-Length: 0

But interestingly appending / at the end:
curl -I http://localhost:8080/ords/
HTTP/1.1 404 Not Found
Error-Reason: error="request"; error_description*=UTF-8''The%20request%20could%20not%20be%20mapped%20to%20any%20database.%20Check%20the%20request%20URL%20is%20correct%2c%20and%20that%20URL%20to%20database%20mappings%20have%20been%20correctly%20configured
Content-Length: 0

**Comment by fuzziebrain** [comment]
@rcordoni I edited your comments and masked your email address. Are you able to capture the Docker logs and upload it here? Also, what host operating system are you? Thanks.

**Comment by rcordoni** [comment]
I managed to get it to work by deleting my oradata folder and re-running with a few minor changes that I included from a previous attempt I had tried months ago using the 18c SE docker image.  I don't think these minor changes had an effect - but this is what I changed the docker run to:
docker run -d --name $CONTAINER_NAME --hostname $CONTAINER_NAME \
        -p ${DOCKER_ORDS_PORT:-50080}:8080 \
        -p ${DOCKER_EM_PORT:-55500}:5500 \
        -p ${DOCKER_DB_PORT:-51521}:1521 \
        --env-file $ENV_FILE \
        -v ~/data/oradata:/opt/oracle/oradata \
        -v $PWD/scripts/setup:/opt/oracle/scripts/setup \
        -v $PWD/scripts/startup:/opt/oracle/scripts/startup \
        -v $PWD/files:/tmp/files \
        -e TZ="America/Vancouver" \
        -e ORACLE_ALLOW_REMOTE=true \
        oracle/database:${DB_VERSION}-${DB_EDITION}

Minor changes were:
- Set hostname of the container
- Set timezone to my timezone
-  Change location of my oradata folder on my local system (Mac).  Also commented out the chown in your scripts as this caused a problem accessing the oradata folder - I made all ugo=rwx instead of changing ownership.
- Added environment variable ORACLE_ALLOW_REMOTE = true - I can't remember where I got this workaround - probably doesn't do anything ...

Anyways - it is working now.

I really appreciate the work you put into this.  It is the only docker config that "everything" painlessly for Oracle Apex on XE.
